apps/internal/base/base.go

```diff
@@ -302,3 +306,30 @@ func convertStrUnixToUTCTime(unixTime string) (time.Time, error) {
 	}
 	return time.Unix(timeInt, 0).UTC(), nil
 }
+
+func getKeyFromRequest(authParams authority.AuthParams) string {
+	if authParams.AuthorizationType == authority.ATClientCredentials {
```

Comment:
There is also the OBO case

Reply:
Thanks for the review Bogdan ! 
We don't support OBO flow in MSAL Go yet 
The second one is interesting, we just have GetAccounts() in the library right now, may be discuss little bit more about this one offline ? Have some questions.

Reply:
I have exposed `GetAccount(home_accountID)` equivalent in the latest commit

Reply:
I have scheduled a meeting to discuss if we need GetAccounts() in MSAL Go.